### PR TITLE
Added Resources for C++

### PR DESCRIPTION
**Pull request by ashusengar**
1. Added DSA + interview course for C++
2. Added Free resource to learn C++

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,7 +274,6 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Introduction to Git and Github - Launch School](https://launchschool.com/books/git)
 - [Git Immersion](https://gitimmersion.com/) 
 - [30SecondsOfCode Git Snippets](https://www.30secondsofcode.org/git/p/1)
-- [Git - the simple guide](https://rogerdudler.github.io/git-guide/)
```
Comment (shahednasser) [inline README.md:276]: I'm not sure if you've accidentally removed this but please add it again. You can find it in the main repository.

Comment (ashusengar) [reply README.md:276]: I had removed it accidentally but now I have committed my changes again. Please do the needful.

File: README.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Introduction to Git and Github - Launch School](https://launchschool.com/books/git)
 - [Git Immersion](https://gitimmersion.com/) 
 - [30SecondsOfCode Git Snippets](https://www.30secondsofcode.org/git/p/1)
-
+- [Git - the simple guide](https://rogerdudler.github.io/git-guide/)
```
Comment (shahednasser) [inline README.md:277]: Please add a new line below the resource

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -212,6 +212,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Sparse Table](https://cp-algorithms.com/data_structures/sparse-table.html)
 - [DSA + Java Interview Preparation Course](https://youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
 - [DSA + C++ Interview Preparation Course](https://www.youtube.com/watch?v=z9bZufPHFLU&list=PLfqMhTWNBTe0b2nM6JHVCnAkhQRGiZMSJ)
+-
```
Comment (shahednasser) [inline README.md:215]: Please remove the `-` just add an empty new line. The same for the other new line you've added

Comment (shahednasser) [reply README.md:215]: You still haven't resolved this

File: README.md (hunk 2 above)
Comment (shahednasser) [reply README.md:278]: You still haven't resolved this

File: README.md [hunk 4]
```diff
@@ -594,8 +595,11 @@ To add a resource or contribute, please check out our contribution guide [here](
 ### Programming Courses Free Online
 
 - [SoloLearn](https://www.sololearn.com/learning)
+- [Learn C++](https://www.codecademy.com/learn/learn-c-plus-plus)
+-
```
Comment (shahednasser) [inline README.md:599]: Please remove this dash

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Hello @ashusengar it seems like you're not noticing most of the comments I'm leaving. Let me know if something sounds confusing or you need help with anything.

**Comment by ashusengar** [comment]
Hey @shahednasser, As you had asked to me remove (-) from below the resource and a line below the resource. I did that and have re-checked the same. Let me know it there is anything else.
image

File: README.md (hunk 4 above)
Comment (ashusengar) [reply README.md]: @shahednasser I have removed the dash

File: README.md [hunk 5]
```diff
@@ -274,7 +276,6 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Git Immersion](https://gitimmersion.com/) 
 - [30SecondsOfCode Git Snippets](https://www.30secondsofcode.org/git/p/1)
 - [Git - the simple guide](https://rogerdudler.github.io/git-guide/)
```
Comment (shahednasser) [inline README.md:278]: Thank you for making all the changes @ashusengar but please add a new line here (without the dash, just an empty new line).

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Great thank you!

**Pull request merged**